docs/changes.rst

```diff
@@ -1,23 +1,32 @@
 Changelog
 *********
 
+1.5.1
+=====
+
+* Added ``S3FSStore``. This is a drop-in replacement
+  for the ``Boto3Store`` and will replace it in the next major release.
+
 1.5.0
 =====
 
 * Added concept for closable stores.
+
```

Comment:
I'm surprised about the formatting changes in this file. Did a pre-commit hook do this?

Reply:
No I did this manually after realizing that some of the formatting looked bad. Not related to this PR, moved it to a separate one.